internal/pkg/service/common/configmap/visit.go

```diff
@@ -131,12 +133,18 @@ func doVisit(vc *VisitContext, cfg VisitConfig) error {
 			// Mark field and all its children as sensitive according to the tag
 			field.Sensitive = vc.Sensitive || field.StructField.Tag.Get(sensitiveTag) == "true"
 
-			// Set usage from the tag, or use parent usage text
+			// Set usage from the tag, or use parent value
 			field.Usage = vc.Usage
 			if usage := field.StructField.Tag.Get(configUsageTag); usage != "" {
 				field.Usage = usage
 			}
```

Comment:
Implementation has been partially here, but the code for `shorthand` was missing.